src/main/java/Deadline.java

```diff
@@ -0,0 +1,41 @@
+//package duke;
+/**
+ * Deadline <- Task
+ */
```

Comment:
Is there a better way to describe the Deadline class in the Javadoc comment? I understand what you are trying to convey but perhaps it would be better to elaborate on it! I personally prefer this representation as its very concise however it might not be intuitive enough for other readers.